staging/src/k8s.io/kube-apiextensions-server/test/integration/basic_test.go

```diff
@@ -186,3 +188,204 @@ func TestSimpleCRUD(t *testing.T) {
 		t.Errorf("missing watch event")
 	}
 }
+
+func TestNonNamespaced(t *testing.T) {
```

Comment:
I guess there is quite some overlap with TestSimpleCRUD. Can you try to factor out the equal pieces?